### ISSUE TITLE
Newly created instances wrongly assigned to weak pointer if target is a protocol type

### ISSUE THREAD

**Issue by AlexanderJ** [Swiftify-Corp/Swiftify#217]
If I create a new instance and assign it to a protocol variable id<Protocol> then its Swift equivalent is a weak var instead of the expected let. Example:

https://swiftify.me/1a7f2b

**Comment by AlexanderJ** [comment]
IMHO its worse than the other issue. In the other case we have a class attribute. Here it's a local variable. The resulting swift will not even compile as the newly created instance has no owner here. So in this case there are no exceptions were a weak var would be ok.

**Comment by alex-swiftify** [comment]
@AlexanderJ Fixed along with #202 (will be available after the next conversion update).

Converter output:
```
class Test {
    func uploadFile(atURL URLs: [URL]?, withMessage message: String?) {
        (URLs as NSArray?)?.enumerateObjects({ [self] URL, idx, stop in
            let path = URL.path
            let uploadFileModel = PREConverseChatUploadFileModel(caseId: caseId, dataId: UUID().uuidString, path: path)
        })
    }
}
```
